sa/storage-authority.go

```diff
@@ -636,3 +636,23 @@ func (ssa *SQLStorageAuthority) AlreadyDeniedCSR(names []string) (already bool,
 
 	return
 }
+
+// GetSCTReceipt gets a specific SCT receipt for a given certificate serial and
+// CT log ID
+func (ssa *SQLStorageAuthority) GetSCTReceipt(serial string, logID string) (receipt core.SignedCertificateTimestamp, err error) {
+	err = ssa.dbMap.SelectOne(
```

Comment:
So this means that folks have to check sql.ErrNoRows which is a sql thing and maybe one day we have to cache this data.

Instead, let's check that for them and return an explicit error type for this. It can be very simple.